.test/config-simple/config.yml

```
@@ -64,40 +64,49 @@ min_gene_expr: 10
-#
-# the heatmap color map
-# see https://seaborn.pydata.org/tutorial/color_palettes.htm for an overview
-colormap: "flare"
+
+# This section defines the pyDESeq2 plot and data handling parameters
+deseq2:
+    # normalization fit type, must be 'parametric' or 'mean'
+    fit_type: "mean"
+    # the "design factors" are the confounding variables to be adjusted fr
+    # during normalization. They must be given in the configuration (samples.csv).
+    design_factors: 
```

yamllint

[error] trailing spaces

(trailing-spaces)